app/bundles/CoreBundle/Tests/Unit/IpLookup/IpapiLookupTest.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace Mautic\CoreBundle\Tests\Unit\IpLookup;
+
+use GuzzleHttp\Client;
+use GuzzleHttp\Psr7\Response;
+use Mautic\CoreBundle\IpLookup\IpapiLookup;
+
+class IpapiLookupTest extends \PHPUnit\Framework\TestCase
+{
+	private $cacheDir = __DIR__.'/../../../../../../var/cache/test';
+
+	public function testIpLookupSuccessful()
+	{
+		// Mock http connector
+		$mockHttp = $this->createMock(C`lient::class);`
```

Comment:
How did this happen? 🙂 
```suggestion
		$mockHttp = $this->createMock(Client::class);
```